src/common/JSHandle.ts

```diff
@@ -416,6 +416,84 @@ export class ElementHandle<
     return result;
   }
 
+  /**
+   * Wait for the `xpath` within the element. If at the moment of calling the
+   * method the `xpath` already exists, the method will return immediately. If
+   * the `xpath` doesn't appear after the `timeout` milliseconds of waiting, the
+   * function will throw.
+   *
+   * If `xpath` starts with `//` instead of `.//`, the dot will be appended automatically.
```

Comment:
`//` means querying from the root node so we cannot allow `//` at it might return elements outside of the element's subtree. So I think this comment is useful to indicate that we rewrite `//` as `.//`.

Reply:
Maybe I could phrase it better 🤔

Reply:
It's just a nit. Feel free to move forward with this.